R/RebuildPlot.R

```diff
@@ -35,9 +35,8 @@
 #' Default="default".
 #' @param pdf Option to send figures to pdf file instead of plot window in
 #' Rgui. Default=FALSE.
-#' @param pwidth Width of the plot window or PDF file (in inches). Default=7.
-#' @param pheight Height of the plot window or PDF file (in inches). Default=7.
-#' @param lwd Line width for many of the plot elements. Default=2.
+#' @template pwidth_pheight
```

Comment:
I see that @k-doering-NOAA and @iantaylor-NOAA started a conversation about the use of a single file for multiple arguments already but from that conversation I did not expect this behaviour. I thought that pwidth_pheight.R would contain
```
@param pwidth, pheight <Description for both parameters>
```
and not two separate calls to `@param`. I think that we should make them two separate files.

Reply:
Ok, I'll change it back! I made this change before starting that conversations, but was too lazy to change it back.

As a side note, is it possible to have description for both parameters in a single line? I've never seen that!

Reply:
I separated them into 2 templates.